twitter/app/Http/Controllers/TweetController.php

```diff
@@ -18,11 +20,16 @@ public function index(Request $request): View
     {
         $tweet = new Tweet();
         $keyword = $request->input('keyword');
-        if(!empty($keyword)) {
+        if (!empty($keyword)) {
             $allTweets = $tweet->searchByQuery($keyword);
         } else {
             $allTweets = $tweet->getAllTweets();
         }
```

Comment:
mustじゃないけど、こんな書き方もできるんだよーって紹介！
```suggestion
            $allTweets = !empty($keyword) ? $tweet->searchByQuery($keyword) : $tweet->getAllTweets();
```